CRM/Price/Form/Field.php

```diff
@@ -308,7 +308,7 @@ public function buildQuickForm() {
       // is active ?
       $this->add('checkbox', 'option_status[' . $i . ']', ts('Active?'));
 
-      $this->add('select', 'option_visibility_id[' . $i . ']', ts('Visibility'), array('' => ts('- select -')) + $visibilityType);
+      $this->add('select', 'option_visibility_id[' . $i . ']', ts('Visibility'), $visibilityType);
```

Comment:
@MiyaNoctem if this '' placeholder is removed aren't you making it mandatory? Because then ```visibility``` field will be set to option which comes first

Reply:
@monishdeb I think it should be mandatory - it is either 'Public' or 'Admin', it can't be unselected because then we don't know who to display it to. It is currently mandatory on the form when editing an existing price option.

Reply:
Hmm makes sense :)